### DISCUSSION TITLE
Swagger UI & Vercel: Unexpected token < in JSON at position 1

### DISCUSSION THREAD

**Discussion by zopagaduanjr** [vercel/community#1772] [Help] [answered]
https://stackoverflow.com/questions/69146517/swagger-ui-vercel-unexpected-token-in-json-at-position-1

Still exists with the following:
express: 4.17.3
swagger-ui-dist: 4.6.2
swagger-ui-express: 4.3.0
swagger-jsdoc: 6.1.0

**Comment by Hazelwu2** [comment]
I have same question.

**Reply by Whisker17** [reply]
Do you have any ideas on how to fix it?

**Comment by amyegan** [comment, marked as answer]
It sounds like the request is being rejected by the API and returning HTML. The most common reason I can think of is that you might be missing an environment variable for your API access token. But there are many possible causes, and finding the right answer will depend on your particular code.

If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look.
